src/store/index.js

```diff
@@ -5,19 +5,30 @@ import { INITIALIZE_PNP } from './action-types.js'
 import { UPDATE_AVAILABLE } from './mutation-types'
 
 Vue.use(Vuex)
+let edgeVersion = require('@/../package.json').version
+
+fetch('http://localhost:8778/api/status')
```

Comment:
@vickywane this function calling the status API belongs in the EdgeAPI class:

https://github.com/ambianic/ambianic-ui/blob/master/src/remote/edgeAPI.js